### PR TITLE
Fixed ServerActiveDirectoryAdministratorTest

### PR DESCRIPTION
**Pull request by j82w**
Removed a check for the graph endpoint when setting the Azure SQL Server Active Directory administrator that caused the test to fail.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@j82w__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jawilley). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs [hunk 1]
```diff
@@ -65,10 +65,6 @@ public MicrosoftAzureCommandsResourcesModelsActiveDirectory.ActiveDirectoryClien
             {
                 if (_activeDirectoryClient == null)
                 {
-                    if (!Profile.Context.Environment.IsEndpointSet(AzureEnvironment.Endpoint.Graph))
```
Comment (akromm) [inline src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs]: Will the test succeeded if the graph endpoint is not set?

**Comment by akromm** [comment]
Looks good to me.

**Comment by azurecla** [comment]
Hi __@j82w__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jawilley). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs (hunk 1 above)
Comment (markcowl) [inline src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs]: Is this a safe change?  Do you need the graph endpoint in one of your clients?

Comment (j82w) [inline src/ResourceManager/Sql/Commands.Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs]: I added the check back in with the latest commit, and set the graph endpoint in the test framework.

**Pull request merged**